src/getty-generator/getty-generator.c

```diff
@@ -31,7 +31,7 @@ static int add_getty_impl(const char *tty, const char *type, const char *unit_pa
         assert(unit_path);
 
         if (!filename_is_valid(tty)) {
-                log_debug("Invalid %s tty device specified, ignoring: %s", type, tty);
+                log_debug("Invalid %s tty device specified, ignoring: %s", type, strna(tty));
```

Comment:
This seems iffy. If `tty==NULL` here, then this is because we internally called `path_startswith()` and got NULL, not something that is the user's fault. So the message with "n/a" would be meaningless. Instead, `add_container_getty` should skip the call to `add_getty_impl` if it sets `tty=NULL`.

Reply:
OK. Now the original path is logged.